tests/optim/models/test_models.py

```diff
@@ -58,7 +53,7 @@ def test_load_inceptionv1_no_redirected_relu(self) -> None:
             )
         model = googlenet(pretrained=True, replace_relus_with_redirectedrelu=False)
         _check_layer_not_in_model(self, model, RedirectedReluLayer)
-        _check_layer_in_model(self, model, ReluLayer)
+        _check_layer_in_model(self, model, torch.nn.ReLU)

```

Comment:
I think that `_check_layer_in_model` and `_check_layer_not_in_model` could be moved to a helper file. That way we can also use it in the `TestReplaceLayers` test in `test_models_common.py`, instead duplicating the function with the `check_is_not_instance` function.

Reply:
@NarineK I'm not though if we should move it an optim test helper file, or one of the more general Captum test helper files?

Reply:
I think for the moment we could just put them a helper file inside the optim module's tests.